src/Drivers/DatabaseDriver.php

```diff
@@ -265,8 +271,8 @@ protected function insert($feature, $scope, $value)
             'name' => $feature,
             'scope' => Feature::serializeScope($scope),
             'value' => json_encode($value, flags: JSON_THROW_ON_ERROR),
-            'created_at' => Carbon::now(),
-            'updated_at' => Carbon::now(),
+            'created_at' => $now = Carbon::now(),
+            'updated_at' => $now,
```

Comment:
This ensures that the same timestamp is uses for both values. Extremely small, but real possibility that they are different.